lib/compass-rails.rb

```diff
@@ -35,14 +35,14 @@ def sass_config
     end
 
     def sprockets
-      @sprockets ||= ::Rails.application.assets
+      @sprockets ||= ::Rails.application.assets || ::Rails.application.config.assets
```

Comment:
`Rails.application.assets` and `Rails.application.config.assets` are _completely different_ things. They are not mutually interchangeable.